src/sentry/static/sentry/app/components/stream/group.jsx

```diff
@@ -123,14 +177,85 @@ const StreamGroup = createReactClass({
         {hasGuideAnchor && <GuideAnchor target="issue_stream" />}
         {withChart && (
           <Box width={160} mx={2} className="hidden-xs hidden-sm">
-            <GroupChart id={data.id} statsPeriod={statsPeriod} data={data} />
+            <GroupChart
+              id={data.id}
+              statsPeriod={statsPeriod}
+              data={data}
+              hasDynamicIssueCounts={hasDynamicIssueCounts}
+            />
           </Box>
         )}
         <Flex width={[40, 60, 80, 80]} mx={2} justifyContent="flex-end">
-          <StyledCount value={data.count} />
+          <Tooltip
+            disabled={!hasDynamicIssueCounts}
+            popperStyle={popperStyle}
+            tipContent={
+              <TooltipContent>
+                {data.lifetime && (
+                  <TooltipRow>
+                    <TooltipText>Events since issue began</TooltipText>
+                    <TooltipCount>{data.lifetime.count}</TooltipCount>
+                  </TooltipRow>
+                )}
+                <TooltipRow>
+                  <TooltipText>Events within {summary}</TooltipText>
+                  <TooltipCount>{data.count}</TooltipCount>
+                  <StyledIconTelescope color={theme.blue300} />
+                </TooltipRow>
+                {data.filtered && (
+                  <TooltipRow>
+                    <TooltipText>Events with filters applied</TooltipText>
+                    <TooltipCount>{data.filtered.count}</TooltipCount>
+                    <StyledIconTelescope color={theme.blue300} />
+                  </TooltipRow>
+                )}
+              </TooltipContent>
+            }
+          >
+            <StyledPrimaryCount value={primaryCount} />
+            {showLifetimeStats && secondaryCount && (
+              <React.Fragment>
+                {'/'}
+                <StyledSecondaryCount value={secondaryCount} />
```

Comment:
This is a nit/opinion, but you could handle this in css using `content` as the separator can be considered part of the "design". I don't know how @robinrendle would feel about this though.

Reply:
I'll do it